README.md

```diff
@@ -19,3 +19,53 @@ To make a release:
 * scroll to the bottom, select latest version, and click `Release`
 * `git tag vx.y`
 * `git push origin vx.y`
+
+## Speller: `pos_tagger_scripts` (legacy)
+Before Q1 2024, the entire binary build and release process relied on Grammalecte data and legacy scripts.
+Some modifications were implemented in the legacy scripts, and these changes will be detailed.
+The Part-of-Speech (PoS) tagging logic is outlined below. 
+However, with the discontinuation of the Grammalecte project in December 2022, there will be no new dicollecte releases. 
+Consequently, the steps preceding Step 4 are no longer necessary for replication.
+
+### Main Script: `Build_LT`
+The main script, named `Build_LT`, orchestrates the entire process step by step with added debugging functionalities. At each stage, the script echoes the current step, and each step outputs a file with a unique name, facilitating easy tracking and debugging. This was not as readable in the original state of the repo.
+
+
+**Dependencies:**
+- Grammalectv7.0 data
+- French frequency words list (both legacy and already present in the repository)
```

Comment:
🔍 This is not a dependency of this script, though.

Reply:
Ah, true, would "Sources" be better?

Reply:
I don't think that's used as a source, either... The frequency word list is only used in the Morfologik compilation step. We can prob. just delete this line.

Also, as far as dependencies go, there's also `Perl`.